### PR TITLE
adding a low code and move Buidbase to it, add appsmith, and add tooljet

### PR DESCRIPTION
**Pull request by mabujaber**
I contributed before and added budibase under the BaaS but it doesn't make sense to be under that section and tried to find another section that meets low-code platform but didn't find any. 

So, I created a new section and move Buidbase and add another service called appsmith, and tooljet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,61 +15,64 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
...
-   * [Visitor Session Recording](#visitor-session-recording)
-   * [Web Hosting](#web-hosting)
-   * [Commenting Platforms](#commenting-platforms)
-   * [Browser based hardware emulation](#browser-based-hardware-emulation-written-in-javascript)
-   * [Remote Desktop Tools](#remote-desktop-tools)
-   * [Game Development](#game-development)
-   * [Other Free Resources](#other-free-resources)
+- [free-for.dev](#free-fordev)
+- [Table of Contents](#table-of-contents)
+  - [Major Cloud Providers](#major-cloud-providers)
```
Comment (ripienaar) [inline README.md:20]: Please restrict this to just adding what you need. We like the markdown syntax we use and general layout here.

File: README.md [hunk 2]
```diff
@@ -15,61 +15,64 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
...
-   * [Vagrant Related](#vagrant-related)
-   * [Visitor Session Recording](#visitor-session-recording)
-   * [Web Hosting](#web-hosting)
-   * [Commenting Platforms](#commenting-platforms)
-   * [Browser based hardware emulation](#browser-based-hardware-emulation-written-in-javascript)
-   * [Remote Desktop Tools](#remote-desktop-tools)
-   * [Game Development](#game-development)
-   * [Other Free Resources](#other-free-resources)
+- [free-for.dev](#free-fordev)
+- [Table of Contents](#table-of-contents)
```
Comment (ripienaar) [inline README.md:19]: also dont need these 2 here, seems redundant

File: README.md [hunk 3]
```diff
@@ -1032,10 +1035,14 @@ Table of Contents
   * [tyk.io](https://tyk.io/) — API management with authentication, quotas, monitoring and analytics. Free cloud offering
   * [zapier.com](https://zapier.com/) — Connect the apps you use, to automate tasks. 5 zaps, every 15 minutes and 100 tasks/month
   * [LeanCloud](https://leancloud.app/) — Mobile backend. 1GB of data storage, 256MB instance, 3K API requests/day, 10K pushes/day are free. (API is very similar to Parse Platform)
-  * [BudiBase](https://budibase.com/) - Budibase is an open-source low-code platform for creating internal apps in minutes. Supports PostgreSQL, MySQL, MSSQL, MongoDB, Rest API, Docker, K8s
 
 **[⬆ back to top](#table-of-contents)**
 
+## Low Code Platform
```
Comment (ripienaar) [inline README.md:1041]: Please format same as other sections

File: README.md (hunk 2 above)
Comment (mabujaber) [reply README.md:19]: Looks like it been generated automatically

File: README.md (hunk 3 above)
Comment (mabujaber) [reply README.md:1041]: sure

File: README.md (hunk 2 above)
Comment (ripienaar) [reply README.md:19]: that would be something in your editor generating it.  There's not automatic index generation.

**Comment by mabujaber** [comment]
I will close this and create another PR

**Pull request closed without merging**